### PR TITLE
W-14569061 - adding usage info. Also A11Y changes: alt text quotes and removing captions

### PR DESCRIPTION
**Pull request by hannanelson**
…ving captions.

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 1]
```diff
@@ -217,30 +217,30 @@ To configure a Mule 3.9 app in Studio 6.x:
 . Install Object Store Connector:
 .. Click the *Exchange* icon in the toolbar:
 +
-image::os-tutorial-studio6-exchange.png[Exchange icon in the Studio 6.x toolbar]
+image::os-tutorial-studio6-exchange.png["Exchange icon in the Studio 6.x toolbar"]
 .. Enter your Anypoint Platform username and password, if prompted.
 .. In Exchange, select *Provided by MuleSoft* and *Connectors*, and then search for Object Store Connector:
 +
-image::os-tutorial-exchange.png[Provided by MuleSoft option and search field in Exchange]
+image::os-tutorial-exchange.png["Provided by MuleSoft option and search field in Exchang"]
```
Comment (hannanelson) [inline modules/ROOT/pages/osv2-tutorial.adoc]:
```suggestion
image::os-tutorial-exchange.png["Provided by MuleSoft option and search field in Exchange"]
```

File: modules/ROOT/pages/osv2-usage.adoc [hunk 2]
```diff
@@ -39,22 +39,52 @@ Object Store v2 returns a `429 HTTP Error` response, indicating that it reached
...
++
+Each 100 KB of payload data above 100 KB counts as an additional unit.
 +
 For example, if an app retrieves one key from an object store with a total payload size of 101KB, the operation counts as two units against the quota.
-* For both the `retrieveAll` and `retrieveAllKeys` operations, the size of all key-value pairs in the object store partition counts towards billable units.
+* For both the Retrieve All and Retrieve All Keys operations, the size of all key-value pairs in the object store partition counts towards billable units.
++
+If an app runs a Retrieve All Keys operation to retrieve only the keys in the object store, the billable unit is based on the size of the key-value pairs, not just the keys.
+
+* For each Store operation, such as storing new values or updating existing ones,
```
Comment (GouthamiKondakindi) [inline modules/ROOT/pages/osv2-usage.adoc]: We need to fix this.. size of just the item being inserted counts towards billable units for Store operation

File: modules/ROOT/pages/osv2-usage.adoc [hunk 3]
```diff
@@ -39,22 +39,52 @@ Object Store v2 returns a `429 HTTP Error` response, indicating that it reached
...
+the size of all key-value pairs in the object store partition counts towards billable units.
++
+For example, if an app inserts or updates one key in an object store with a total payload size of 500KB, the request counts as five units against the quota.
++
+Each 100 KB of payload data above 100 KB counts as an additional unit.
++
+For example, if an app inserts one key in an object store with a total payload size of 101KB, the operation counts as two units against the quota.
```
Comment (GouthamiKondakindi) [inline modules/ROOT/pages/osv2-usage.adoc]: Line 64 and 68 look similar to me. Is there a difference?

Comment (hannanelson) [reply modules/ROOT/pages/osv2-usage.adoc]: Line 68 has a different payload size - 101KB, which pushes it over the payload size and adds a unit.

Comment (hannanelson) [reply modules/ROOT/pages/osv2-usage.adoc]: Rejiggered.

**Review by GouthamiKondakindi — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/osv2-usage.adoc [hunk 4]
```diff
@@ -39,22 +39,57 @@ Object Store v2 returns a `429 HTTP Error` response, indicating that it reached
...
 
-* For each API request, up to 100 KB of object-payload data (if applicable) counts as one unit against the quota.
+* Each API request up to 100 KB of object-payload data (if applicable) counts as one unit against the quota.
 +
 For example, if an app retrieves one key from an object store with a total payload size of 500KB, the request counts as five units against the quota.
-* Each 100 KB of payload data above 100KB counts as an additional unit.
+
+* Each 100 KB of payload data above 100 KB counts as an additional unit.
++
+For example, if an app inserts, updates or retrieves an item  with a total payload size of 101KB from an object store, the operation counts as two units against the quota.
```
Comment (tasmoore) [inline modules/ROOT/pages/osv2-usage.adoc]:
```suggestion
For example, if an app inserts, updates, or retrieves an item with a total payload size of 101KB from an object store, the operation counts as two units against the quota.
```

File: modules/ROOT/pages/osv2-usage.adoc [hunk 5]
```diff
@@ -39,22 +39,57 @@ Object Store v2 returns a `429 HTTP Error` response, indicating that it reached
...
+
+* For the Retrieve All and Retrieve All Keys operations, the size of all key-value pairs in the object store partition counts towards billable units.
```
Comment (tasmoore) [inline modules/ROOT/pages/osv2-usage.adoc]:
```suggestion
* For the Retrieve All and Retrieve All Keys operations, the size of all key-value pairs in the object store partition counts toward billable units.
```

File: modules/ROOT/pages/osv2-usage.adoc [hunk 6]
```diff
@@ -39,22 +39,57 @@ Object Store v2 returns a `429 HTTP Error` response, indicating that it reached
...
 +
-For example, if an app retrieves one key from an object store with a total payload size of 101KB, the operation counts as two units against the quota.
-* For both the `retrieveAll` and `retrieveAllKeys` operations, the size of all key-value pairs in the object store partition counts towards billable units.
+For example, if an app runs the Retrieve All Keys operation to retrieve only the keys in the object store, the billable unit is based on the size of the key-value pairs, not just the keys.
+
+* For each Store operation, the size of the item being inserted or updated counts towards billable units.
```
Comment (tasmoore) [inline modules/ROOT/pages/osv2-usage.adoc]:
```suggestion
* For each Store operation, the size of the item being inserted or updated counts toward billable units.
```

File: modules/ROOT/pages/osv2-usage.adoc [hunk 7]
```diff
@@ -39,22 +39,57 @@ Object Store v2 returns a `429 HTTP Error` response, indicating that it reached
 If the store reaches the TPS limit, you can either upgrade to a premium subscription or redesign the app to send API requests more slowly.
 
 
+[[how-billing-is-calculated]]
 == How Billing is Calculated
 
 Effective API requests count towards your monthly billing usage and quota.
```
Comment (tasmoore) [inline modules/ROOT/pages/osv2-usage.adoc]:
```suggestion
Effective API requests count toward your monthly billing usage and quota.
```

**Review by tasmoore — DISMISSED** [review]

**Review by tasmoore — APPROVED** [review]

**Pull request merged**